crates/symbolicator/src/services/download/mod.rs

```diff
@@ -65,10 +68,26 @@ impl DownloadError {
             DownloadError::Gcs(inner) => format!("{}: {}", self, inner),
             DownloadError::Sentry(inner) => format!("{}: {}", self, inner),
             DownloadError::S3(inner) => format!("{}: {}", self, inner),
+            DownloadError::Permissions => String::from("insufficient permissions"),
```

Comment:
we can just use the string representation of `Download::Permissions` here, any micro-optimization achieved by using this marginally shorter string isn't really worth it, and we no longer need to track yet another magic string by removing `"insufficient permissions"`